internal/cmd/local/command_kafka_topic_consume.go

```diff
@@ -0,0 +1,163 @@
+package local
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+
+	ckafka "github.com/confluentinc/confluent-kafka-go/kafka"
+
+	"github.com/confluentinc/cli/internal/cmd/kafka"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newConsumeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "consume <topic>",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.topicConsume,
+		Short: "Consume messages from a Kafka topic.",
+		Long:  "Consume messages from a Kafka topic. Configuration and command guide: https://docs.confluent.io/confluent-cli/current/cp-produce-consume.html.\n\nTruncated message headers will be printed if they exist.",
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Consume message from topic "my_topic" from the beginning and with keys printed.`,
```

Comment:
Nit: Can we avoid mixing symbols in our examples, and just stick with lower-case-with-dashes?
```suggestion
				Text: `Consume message from topic "my-topic" from the beginning and with keys printed.`,
```